psydac/api/feec.py

```diff
@@ -83,6 +92,11 @@ def V2(self):
     def V3(self):
         return self._spaces[3]
 
+    @property
+    def Vvec(self):
+        assert self.has_vec
```

Comment:
docstring missing. What space is this?

Reply:
Yes, I think we should add docstrings for all of these properties. We should have done it before, could you please write one-line docstrings? Please let me know if you have any questions.